sympy/utilities/autowrap.py

```diff
@@ -1063,22 +1082,13 @@ def ufuncify(args, expr, language=None, backend='numpy', tempdir=None,
     >>> f(np.arange(5), 3)
     array([  3.,   4.,   7.,  12.,  19.])
 
-    For the 'f2py' and 'cython' backends, inputs are required to be equal length
-    1-dimensional arrays. The 'f2py' backend will perform type conversion, but
-    the Cython backend will error if the inputs are not of the expected type.
-
-    >>> f_fortran = ufuncify((x, y), y + x**2, backend='f2py')
-    >>> f_fortran(1, 2)
-    array([ 3.])
-    >>> f_fortran(np.array([1, 2, 3]), np.array([1.0, 2.0, 3.0]))
-    array([  2.,   6.,  12.])
-    >>> f_cython = ufuncify((x, y), y + x**2, backend='Cython')
-    >>> f_cython(1, 2)  # doctest: +ELLIPSIS
-    Traceback (most recent call last):
-      ...
-    TypeError: Argument '_x' has incorrect type (expected numpy.ndarray, got int)
-    >>> f_cython(np.array([1.0]), np.array([2.0]))
-    array([ 3.])
```

Comment:
I wouldn't remove any examples, just add more.

Reply:
The purpose of this example you removed is to inform about the type conversion (or lack of).